sensor/tests/replay/replay_test.go

```diff
@@ -212,15 +212,15 @@ func (suite *ReplayEventsSuite) Test_ReplayEvents() {
 			}
 			expectedAlerts := getAlerts(expectedEvents)
 			receivedAlerts := getAlerts(allEvents)
-			assert.Equal(t, len(expectedAlerts), len(receivedAlerts))
+			assert.Equal(t, len(expectedAlerts), len(receivedAlerts), "Not the same number of alerts.\nExpected: %+v\nActual: %+v", expectedAlerts, receivedAlerts)
```

Comment:
nit: use `assert.Len` to assert length of slices. I think the message will be more clear that the number of events don't match. 

Check if using `.Len` would print out the elements of the slice. If so the extra message wouldn't be needed.

Reply:
`.Len` would print only one of the slices. We could do something like: 

```go
assert.Len(t, receivedAlerts, len(expectedAlerts), "Not the same number of alerts. Expected: %+v", expectedAlerts)
```

What do you think?

Reply:
I think that it depends on what you want to assert. After looking at the code again it seems like this seem redundant to the code below? If the expected event isn't found in the list, the loop would print the specific event that is missing. If my understanding is correct, why would we benefit from printing the entire lists if the lengths are different?

Reply:
The only use for the `len` assertions is in the case when we received more alerts/deployments than we expected. I'm no longer sure if that's a case we should test at all. If you agree, then I'll delete the `len` assertions.

Reply:
Sounds good

Reply:
Done